CIP-0000.md

```diff
@@ -0,0 +1,195 @@
+
+Building consensus with the Celo community must adhere to the [Celo Code of Conduct](https://celo.org/code-of-conduct). Further, your proposal is much more likely to build consensus if you it aligns with the [Celo Community Tenets](https://celo.org/community) and furthers the Celo project's mission.
+
+## Ring 0 and Ring 1 CIPs
+For Ring 0-1 CIPs, given that they require Celo’s client implementation to be considered **Final** (see “CIPs Process” below), the CIP author will need to either provide an implementation for the celo-blockchain client or convince clients to implement your CIP.
+
+The best way to get client implementers to review your CIP is to present it on a Celo AllCoreDevs call. You can request to do so by posting a comment linking your CIP on an agenda issue for an upcoming call on the Github repository.
+
+The Celo AllCoreDevs call serve as a way for client implementers and stakeholders to do two things. First, to discuss the technical merits of CIPs. Second, to coordinate CIP implementation for network upgrades. 
+
+These calls generally result in a “[rough consensus](https://en.wikipedia.org/wiki/Rough_consensus)” around what CIPs should be implemented. Rough consensus, first used by IETF to describe how working groups would proceed, rests on the assumptions that CIPs are not contentious enough to cause a network split and that they are technically sound. It also states that 51% of working group voting on a specific action isn't rough consensus, while 99% is better than rough consensus.
+
+_In short, your role as the champion is to write the CIP using the style and format described below, shepherd the discussions in the appropriate forums, and build community consensus around the idea._
+
+## CIP Process
+When we talk about the community from the perspective of the CIP Process, we are talking about all the validators, stakeholders, core-developers, dapp developers, volunteers and active participants involved in the CIP Process.
+
+Rough Consensus, as mentioned in the previous section, is used to measure community sentiment about a particular proposal. Rough Consensus is measured mostly on the All-Core Devs calls, but it can also happen outside of those calls on Github or Celo Forum comments. Rough Consensus will be determined by the Hardfork Coordinator and All-Core Dev Call organizer based on feedback of the network participants during the call. 
+
+
+Following is the process that a successful Ring 2-3 CIP will move along:
+```
+[ WIP ] -> [ DRAFT ] -> [ LAST CALL ] -> [ FINAL ]
+```
+
+Following is the process that a successful Ring 0-1 or non-Standards CIP will move along:
```

Comment:
nit: why put rings 0-1 after 2-3?

Reply:
It's fairly more complex therefore I added it after since some people are happy with proposing changes on 2-3 ring layer.